File: Arrays and Hashing/14-Longest-Common-Prefix/14.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        if(strs.length == 0){
            return "";
        }
        Arrays.sort(strs);
        StringBuilder result = new StringBuilder();
        String first = strs[0];
        String last = strs[strs.length - 1];
        for(int i = 0; i < first.length(); i++){
            if(i < last.length() && first.charAt(i) == last.charAt(i)){
                result.append(first.charAt(i)); 
            }
            else{
                break;
            }
        }
        return result.toString();
    }
}

File: Arrays and Hashing/14-Longest-Common-Prefix/14.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        if(strs.Length == 0){
            return "";
        }
        Array.Sort(strs);
        string result = "";
        string first = strs[0];
        string last = strs[strs.Length - 1];
        for(int i = 0; i < first.Length; i++){
            if(i < last.Length && first[i] == last[i]){
                result += first[i]; 
            }
            else{
                break;
            }
        }
        return result;
    }
}